src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
```

Comment:
## Notebook comments

Leaving notebook related comments because GitHub still doesn't work very well with notebook. 

- For pulling data, do not pull anything that is not included in the GitHub repository. In general, it is also potentially dangerous to leave "path" snippets in the code base. Especially if it is pointing somewhere on your personal machine/workspace.
- For plotting, use the sipmanalyze/plotting.py file for a more uniform plotting style. See the waveform inspection notebook for examples [1]

[1] https://github.com/UMDCMS/sipmanalyze/blob/master/notebooks/waveform_inspect.ipynb